pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -103,7 +103,7 @@ func (m *kubeGenericRuntimeManager) generateLinuxContainerResources(pod *v1.Pod,
 		// NOTE(ehashman): Behaviour is defined in the opencontainers runtime spec:
 		// https://github.com/opencontainers/runtime-spec/blob/1c3f411f041711bbeecf35ff7e93461ea6789220/config-linux.md#memory
 		switch m.memorySwapBehavior {
-		case kubelettypes.UnlimitedSwap:
+		case "", kubelettypes.UnlimitedSwap:
```

Comment:
this looks like a change in behavior and, if so, needs discussion and docs.
In any case, please elaborate in the commit message why this change is beneficial

Reply:
You're right, no need for this behavior change.
Now the "default" case is configuring "NoSwap", as before.

As a bonus, it now covers both v1 and v2 cases.